tmva/sofie/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building TMVA SOFIE package
+############################################################################
+#Author: Sitong An, Lorenzo Moneta 10/03/2011
+
+#sofie is built only if protobuf is found
+
+include_directories(SYSTEM ${Protobuf_INCLUDE_DIRS} ${CMAKE_CURRENT_BINARY_DIR} )
```

Comment:
I believe this line should be removed...Please post me an error you get while building your library without this line.

Reply:
Hi Oksana, this line is needed so that RModelParser can find the generated onnx_proto3.pb.h